packages/api/internal/template-manager/create_template.go

```diff
@@ -64,49 +52,9 @@ func (tm *TemplateManager) CreateTemplate(
 			StartCommand:       startCommand,
 		},
 	})
-	err = utils.UnwrapGRPCError(err)
-	if err != nil {
-		return fmt.Errorf("failed to create template '%s': %w", templateID, err)
-	}
 
-	// Wait for the build to finish and save logs
```

Comment:
`createTemplate` grpc request is no more stream. It will request build and receive result instantly. Build is done in goroutine and checked with `templateBuildStatus` method.